src/main/java/com/google/devtools/build/lib/bazel/commands/ModCommand.java

```diff
@@ -117,6 +137,7 @@ public void editOptions(OptionsParser optionsParser) {
 
   @Override
   public BlazeCommandResult exec(CommandEnvironment env, OptionsParsingResult options) {
+    env.getEventBus().register(this);
```

Comment:
can we make a special event listener for use by the `tidy` command only? and it probably doesn't need to be a member variable?